Tests/Unit/Repository/Resource/PhpcrResourceTest.php

```diff
@@ -20,6 +20,7 @@ class PhpcrResourceTest extends ProphecyTestCase
 
     public function setUp()
     {
+        parent::setUp();
```

Comment:
We can remove these and the `phpunit-prophecy` dependency. The base test case should just extend `PHPUnit_Framework_TestCase`